stress_test_model_loan_book.R

```diff
@@ -547,10 +560,29 @@ for (i in seq(1, nrow(transition_scenarios))) {
       year == start_year,
       technology %in% technologies,
       scenario_geography == scenario_geography_filter
+    ) #%>%
+    # distinct(investor_name, portfolio_name, company_name, ald_sector, technology,
+    #          scenario_geography, year, plan_carsten, plan_sec_carsten, term,
+    #          PD_0)
+
+  financial_data_loans_pd <- financial_data_loans %>%
+    select(company_name, company_id, ald_sector, technology, pd) %>%
+    distinct(across(everything()))
+
+  plan_carsten_loanbook <- plan_carsten_loanbook %>%
+    # ADO 879: removed company id from join, but should be re-introduced later on
+    left_join(financial_data_loans_pd, by = c("company_name", "ald_sector", "technology"))
+  # TODO: what to do with entries that have NAs for pd?
```

Comment:
is this still open?

Reply:
there are a bunch of TODOs commented into various parts of the loans workflow. This was essentially copied here from elsewhere. I would rather address them in dedicated tasks, because I feel they don't really have anything to do with the workflow fundamentally passing all variables to their respective use cases in the same way that the bonds work flow does that. But they should be addressed before the release

Reply:
ok, resolving issue